fineract-accounting/src/main/java/org/apache/fineract/accounting/rule/api/AccountingRuleApiResource.java

```diff
@@ -196,15 +173,13 @@ public String createAccountingRule(@Parameter(hidden = true) final String jsonRe
     @RequestBody(content = @Content(schema = @Schema(implementation = AccountingRuleApiResourceSwagger.PutAccountingRulesRequest.class)))
     @ApiResponses({
             @ApiResponse(responseCode = "200", description = "OK", content = @Content(schema = @Schema(implementation = AccountingRuleApiResourceSwagger.PutAccountingRulesResponse.class))) })
```

Comment:
Kindly validate whether the `CommandProcessingResult` and `AccountingRuleApiResourceSwagger.PutAccountingRulesResponse` fields are matching!

We can consider removing the annotation and the explicit swagger class if it is not matching!